be.telenet.user.QSSTV.yml

```diff
@@ -0,0 +1,69 @@
+app-id: be.telenet.user.QSSTV
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: qsstv
+rename-desktop-file: qsstv.desktop
+rename-icon: qsstv
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - '*.a'
+  - '*.la'
+  - /bin/usr
+modules:
+  - name: hamlib
+    buildsystem: simple
+    build-commands:
+       - ./configure --prefix=/ --libdir=/lib
+       - make -j4
+       - make install DESTDIR=$FLATPAK_DEST
+    sources:
+      - type: archive
+        url: https://github.com/Hamlib/Hamlib/releases/download/3.3/hamlib-3.3.tar.gz
+        sha256: c90b53949c767f049733b442cd6e0a48648b55d99d4df5ef3f852d985f45e880
+  - name: fftw
+    buildsystem: simple
+    build-commands:
+       - ./configure --prefix=/ --libdir=/lib --enable-shared --enable-static --enable-threads
```

Comment:
Prefix `/` is just wrong IMHO. Using `DESTDIR=$FLATPAK_DEST` is probably not what you want. `$FLATPAK_DEST` is the prefix and DESTDIR is something used for staging.